src/komal/Shop.java

```diff
@@ -0,0 +1,49 @@
+package komal;
+
+public class Shop {
+	int maggieQ = 50;
+	int dosaQ = 43;
+	int pouchesQ = 39;
+	int panipuriQ = 43;
+	int masalaQ = 73;
+
+	void outOfstock(int reqdosa, int reqpanipuri) {
```

Comment:
This is not correct way!
Please refer below Code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/167/files